compiler/rustc_passes/src/check_attr.rs

```diff
@@ -1978,12 +1978,16 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
         }
     }
 
-    fn check_used(&self, attrs: &[Attribute], target: Target) {
+    fn check_used(&self, attrs: &[Attribute], target: Target, span: Span) {
```

Comment:
Suggestion: maybe call this `target_span`? I find just `span` a bit confusing here (there are multiple spans).

Reply:
> Suggestion: maybe call this `target_span`? I find just `span` a bit confusing here (there are multiple spans).

Done. Thank you.